assets/javascripts/lib/protocol.js

```diff
@@ -43,7 +43,9 @@ export const ENCRYPT_PROTOCOL_VERSION = 1;
  * @return {Promise}
  */
 export function generateIdentity(version) {
-  version = version || ENCRYPT_PROTOCOL_VERSION;
+  if (version === undefined) {
+    version = ENCRYPT_PROTOCOL_VERSION;
+  }
```

Comment:
Why this change?

Could also probably do: 

```
version ??= ENCRYPT_PROTOCOL_VERSION;
```

Reply:
Because it wasn't possible to generate a version `0` identity since the `||` treats `0` as a falsey value and it gets changed to 1. TIL about `??=` which is more suitable than `||` in this case, thanks!